spec/features/create_honors_college_thesis_spec.rb

```diff
@@ -24,22 +25,43 @@
       Sipity::Workflow.create(name: 'test', allows_access_grant: true, active: true, permission_template_id: permission_template.id)
     end
 
+    let(:academic_unit_sorted_all_options) do
+      [
+          ["Accounting - 1979/1992, 2009/open", "http://opaquenamespace.org/ns/osuAcademicUnits/KMyb2rzG"],
+          ["Animal Sciences - 1984/2013", "http://opaquenamespace.org/ns/osuAcademicUnits/EaDtECbp"],
+      ]
+    end
+    let(:degree_field_sorted_all_options) do
+      [
+          ["Biological and Organic Chemistry - 1944", "http://opaquenamespace.org/ns/osuDegreeFields/0ARiezTD"],
+          ["Biochemistry - 1941, 1944/1952, 1965", "http://opaquenamespace.org/ns/osuDegreeFields/0Kamj8EG"],
+      ]
+    end
+
     before do
       Hyrax::PermissionTemplateAccess.create(permission_template: permission_template, agent_type: 'user', agent_id: user.user_key, access: 'deposit')
       Sipity::WorkflowAction.create(id: 4, name: 'show', workflow_id: workflow.id)
+      allow_any_instance_of(ApplicationHelper).to receive(:select_tag_dates).and_return("")
+      allow_any_instance_of(Hyrax::DefaultForm).to receive(:date_terms).and_return([])
+      allow_any_instance_of(ScholarsArchive::AcademicUnitsService).to receive(:select_sorted_all_options).and_return(academic_unit_sorted_all_options)
+      allow_any_instance_of(ScholarsArchive::AcademicUnitsService).to receive(:select_sorted_current_options).and_return(academic_unit_sorted_all_options)
+      allow_any_instance_of(ScholarsArchive::DegreeFieldService).to receive(:select_sorted_all_options).and_return(degree_field_sorted_all_options)
+      allow_any_instance_of(ScholarsArchive::DegreeFieldService).to receive(:select_sorted_current_options).and_return(degree_field_sorted_all_options)
+#      allow(controller).to receive(:current_user).and_return(current_user)
+
       login_as user
```

Comment:
I think also `login_as user` might not work as expected since we are using CAS authentication. I wonder it would work having  instead `RubyCAS::Filter.fake(user.onid)` like in https://github.com/osulp/Room-Reservation/blob/master/spec/features/admin/room_spec.rb#L8